packages/editor/src/store/defaults.js

```diff
@@ -28,6 +29,7 @@ export const EDITOR_SETTINGS_DEFAULTS = {
 	richEditingEnabled: true,
 	codeEditingEnabled: true,
 	fontLibraryEnabled: true,
+	fontUploadsEnabled: true,
```

Comment:
It's probably safer to assume false as the default if the filter is removed. That will avoid the UI showing only to display an error if the caps check fails.